apps/condo/domains/banking/schema/BankAccount.test.js

```diff
@@ -62,6 +73,20 @@ describe('BankAccount', () => {
                 expect(bankAccount.currencyCode).toEqual('RUB')
             })
 
+            test('service can', async () => {
```

Comment:
"service user can"

Reply:
Not consistent. All tests in the file related to User are written without mentioning the entity. You can see above. ('admin can', 'support can', 'user can')
And also the createservice script creates tests with this naming